packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx

```diff
@@ -92,38 +92,6 @@ describe("Locked Function Settings", () => {
             // eslint-disable-next-line testing-library/no-node-access
             .querySelector("input");
         expect(inputField?.value).toEqual("");
-
-        // Act (max domain not defined)
-        cleanup();
-        render(
-            <LockedFunctionSettings {...defaultProps} domain={[0, null]} />,
-            {
-                wrapper: RenderStateRoot,
-            },
-        );
-
-        // Assert
-        inputField = screen
-            .getByLabelText("domain max")
-            // eslint-disable-next-line testing-library/no-node-access
-            .querySelector("input");
-        expect(inputField?.value).toEqual("");
-
-        // Act (min domain not defined)
-        cleanup();
-        render(
-            <LockedFunctionSettings {...defaultProps} domain={[null, 0]} />,
-            {
-                wrapper: RenderStateRoot,
-            },
-        );
-
-        // Assert
-        inputField = screen
-            .getByText("domain min")
-            // eslint-disable-next-line testing-library/no-node-access
-            .querySelector("input");
-        expect(inputField?.value).toEqual("");
```

Comment:
These tests can go away because the elements of `domain` can no longer be `null` here!